### PR TITLE
feat: expose refetchInterval

### LINKED ISSUE #20: Expose refetchInterval for simple app data synchronization
Expose the `refetchInterval` feature provided by react-query: https://react-query.tanstack.com/reference/useQuery

Is it something to keep ?

### PR DESCRIPTION
**Pull request by swouf**
Closes #20 

Maybe this feature can be temporary and replaced by WebSockets?

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
@swouf, it looks good to me, but @spaenleh will confirm.

**Review by juancarlosfarah — APPROVED** [review]
Looks good to me, but I would wait for another approval from @spaenleh.

**Review by spaenleh — APPROVED** [review]
Looks good to me, as @swouf said, it is a hack for websockets

**Pull request merged**
